build/images/Dockerfile.build.agent.coverage

```diff
@@ -35,10 +35,13 @@ LABEL description="The Docker image to deploy the antrea-agent with code coverag
 
 USER root
 
+RUN mkdir -p /tmp/coverage
 COPY build/images/scripts/* /usr/local/bin/
 COPY --from=antrea-build /antrea/bin/antrea-agent /usr/local/bin/
 COPY --from=antrea-build /antrea/bin/antrea-agent-coverage /usr/local/bin/
 COPY --from=antrea-build /antrea/bin/antrea-cni /usr/local/bin/
 COPY --from=antrea-build /antrea/bin/antctl /usr/local/bin/
 COPY --from=antrea-build /antrea/bin/antctl-coverage /usr/local/bin/
-COPY --from=antrea-build /antrea/test/e2e/coverage/agent-arg-file /
+COPY --from=antrea-build /antrea/test/e2e/utils/run_cov_binary.sh /
+
+ENTRYPOINT [ "/./run_cov_binary.sh" ]
```

Comment:
what purpose does the leading `/` serve?

Reply:
Just to make sure it is run from the root directory, where the script is.

Reply:
Should probably just be `ENTRYPOINT [ "/run_cov_binary.sh" ]` then